CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # 1.0.1 (2017-05-10)
```

Comment:
Date and version seem off.

Reply:
@Blackbaud-BobbyEarl Fixed :) good catch.